ui/app/AppLayouts/Communities/popups/DiscordImportProgressContents.qml

```diff
@@ -364,4 +368,28 @@ StatusScrollView {
             }
         }
     }
+
+    Component {
+        id: deleteAndRestartConfirmationPopupCmp
+        ConfirmationDialog {
+            id: deleteAndRestartConfirmationPopup
+            headerSettings.title: qsTr("Are you sure you want to delete the channel?")
+            confirmationText: qsTr("Your new Status %1 will be deleted and all information entered will be lost.").arg(root.importingSingleChannel ? qsTr("channel") : qsTr("community"))
```

Comment:
This should be just "channel", there's a TODO for "community" above and "Delete channel" below